src/main/java/uk/gov/hmcts/darts/authorisation/controller/impl/AuthorisationControllerImpl.java

```diff
@@ -24,10 +20,7 @@ public class AuthorisationControllerImpl implements AuthorisationController {
     @Override
     public UserState getUserState() {
         UserAccountEntity currentUser = authorisationApi.getCurrentUser();
-        Optional<UserState> userStateOptional = authorisationApi.getAuthorisation(currentUser.getEmailAddress());
-        if (userStateOptional.isEmpty()) {
-            throw new DartsApiException(AuthorisationError.USER_NOT_AUTHORISED_FOR_ENDPOINT);
```

Comment:
Author note: If the user email provided in the JWT does not match an email in the database, then `authorisationApi.getCurrentUser()` will throw an exception, so this line can never be reached.